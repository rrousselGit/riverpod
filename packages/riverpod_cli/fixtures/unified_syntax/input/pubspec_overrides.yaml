dependency_overrides:
  flutter_riverpod: ^0.14.0+1
  hooks_riverpod: ^0.14.0+1
  riverpod: ^0.14.0+1
