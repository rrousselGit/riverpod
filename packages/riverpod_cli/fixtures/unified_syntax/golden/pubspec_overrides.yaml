dependency_overrides:
  flutter_riverpod: ^1.0.4
  hooks_riverpod: ^1.0.4
  riverpod: ^1.0.3+1
